client-base/src/test/kotlin/app/cash/backfila/client/fixedset/FixedSetBackend.kt

```diff
@@ -34,13 +35,16 @@ class FixedSetBackend @Inject constructor(
     }
   }
 
-  private fun <Param : Any> createOperator(
-    backfill: FixedSetBackfill<Param>,
-  ) = FixedSetBackfillOperator(
-    backfill = backfill,
-    datastore = datastore,
-    parametersOperator = BackfilaParametersOperator(parametersClass(backfill::class)),
-  )
+  @Suppress("UNCHECKED_CAST")
```

Comment:
Do we need this? I don't know that I like casting the operator, I'd rather cast the Param type exclusively.